README.md

```diff
@@ -66,9 +66,7 @@ Documentation
 
 Contributing
 ------------------
-* Git Setup: https://github.com/wildfly/wildfly/blob/main/docs/src/main/asciidoc/_hacking/github_setup.adoc
-* Contributing: https://github.com/wildfly/wildfly/blob/main/docs/src/main/asciidoc/_hacking/contributing.adoc
-* Pull request standard: https://github.com/wildfly/wildfly/blob/main/docs/src/main/asciidoc/_hacking/pullrequest_standards.adoc
+Checkout instructions available in the [contribution guide](CONTRIBUTING.md).
```

Comment:
s/Checkout/ Please see the/g

I say this because "check out" is an overloaded term in the context of a git-based software project. The other places you use it the meaning is pretty clear, but here it's not clear.